app/services/service_listeners/publishing_api_html_attachments.rb

```diff
@@ -18,8 +18,20 @@ def publish
     alias_method :unwithdraw, :publish
 
     def republish
-      update_publishing_api_content
-      unpublish_if_required
+      if Edition::PRE_PUBLICATION_STATES.include?(edition.state)
+        update_draft(update_type: "republish")
+      elsif edition.unpublishing && edition.withdrawn?
```

Comment:
Would just `edition.withdrawn?` be sufficient? I don't _think_ you can have a withdrawn edition without an unpublishing... 🤔

Reply:
yup I can apply this